### PR TITLE
docs: some markdown linting

### PR DESCRIPTION
**Pull request by philclifford**
A little pacification of markdownlint.
(NB :  MD030/list-marker-space disabled)

Plus some style changes to make copy-pasta easier and so on.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,33 +34,40 @@ At a high level, this is what Quicktest is all about:
 
 ## Quick Start
 
-**Quicktest** requires [quickemu](https://github.com/quickemu-project/quickemu) (and quickget), `qemu`, `ffmpeg`, `socat`, `convert` (from imagemagick) and `tesseract`
+**Quicktest** requires [quickemu](https://github.com/quickemu-project/quickemu) (and quickget), `qemu`, `ffmpeg`, `socat`, `convert` (from imagemagick) and `tesseract` : e.g.
 
-e.g. `sudo apt install socat ffmpeg imagemagick tesseract`
+``` shell
+sudo apt install socat ffmpeg imagemagick tesseract
```
Comment (philclifford) [inline README.md]: Or
```suggestion
sudo apt install socat ffmpeg imagemagick tesseract-ocr
```
Or not

Comment (popey) [reply README.md]: You're right!

Comment (philclifford) [reply README.md]: or wrong ...  There should have been a button for you to commit that suggestion to my branch if you liked it btw.

**Comment by philclifford** [comment]
Still whines about MD036

https://github.com/DavidAnson/markdownlint/blob/v0.34.0/doc/md036.md

unless either:
- MD036 is disabled
OR
- those emphasised lines are made structural

**Pull request merged**